File: Java/BubbleSort.java
import java.util.Scanner;

class BubbleSort {
	
	//Bubble sort in ascending order
	void bubblesort(int arr[])
	{		
		for(int i=0;i<arr.length-1;i++)
		{
			for(int j=0;j<arr.length-i-1;j++)
				if(arr[j]>arr[j+1])
				{
					int tmp=arr[j];
					arr[j]=arr[j+1];
					arr[j+1]=tmp;
				}
		}
		
		//printing the sorted array
		for(int i=0;i<arr.length;i++)
			System.out.print(arr[i]+" ");
	}
	//Main function taking input from user
	public static void main(String[] args) {
		
		Scanner input=new Scanner(System.in);
		BubbleSort obj=new BubbleSort();
		
		System.out.print("Enter number of elements in array:");
		int n=input.nextInt();
		
		int[] arr=new int[n];
		System.out.println("Enter elements of the array:");
		for(int i=0;i<n;i++)
		arr[i]=input.nextInt();
		
		System.out.println("Sorted array:");
		obj.bubblesort(arr);	
	}
}

File: Kotlin/BubbleSort.kt
import java.util.Scanner

class BubbleSort {

    //Bubble sort in ascending order
    fun bubblesort(arr: IntArray) {
        for (i in 0 until arr.size - 1) {
            for (j in 0 until arr.size - i - 1)
                if (arr[j] > arr[j + 1]) {
                    val tmp = arr[j]
                    arr[j] = arr[j + 1]
                    arr[j + 1] = tmp
                }
        }

        //printing the sorted array
        for (i in arr.indices)
            System.out.print(arr[i].toString() + " ")
    }

    companion object {
        //Main function taking input from user
        fun main(args: Array<String>) {

            val input = Scanner(System.`in`)
            val obj = BubbleSort()

            System.out.print("Enter number of elements in array:")
            val n = input.nextInt()

            val arr = IntArray(n)
            System.out.println("Enter elements of the array:")
            for (i in 0 until n)
                arr[i] = input.nextInt()

            System.out.println("Sorted array:")
            obj.bubblesort(arr)
        }
    }
}
